packages/atomic/src/components/common/generated-answer/generated-content/markdown-utils.test.ts

```diff
@@ -302,6 +302,22 @@ describe('markdownUtils', () => {
           );
         });
       });
+
+      it('should transform formatted heading', () => {
+        const text = '# **bold** *emphasized* with `code` title';
+        const html = transformMarkdownToHtml(text);
+        expect(removeLineBreaks(html)).toBe(
+          '<div part="answer-heading-1" aria-label="bold emphasized with code title"><strong part="answer-strong">bold</strong> <em part="answer-emphasis">emphasized</em> with <code part="answer-inline-code">code</code> title</div>'
+        );
+      });
+
+      it('should transform heading with nested formatting', () => {
```

Comment:
should we also have this test for nested formatting in the quantic tests?

Reply:
Considering the test suite was simplified for Quantic, I went with a single test to cover the change. I don't think adding another test for nested formatting is necessary.